docs/supported_backups.md

```diff
@@ -0,0 +1,30 @@
+# OADP Supported Backups
+
+OADP supports backup and restore disaster recovery operations for both containerized and OpenShift Virtualization workloads. It is important to distinguish the nuances and support for each workload type.
+
+## Containerized Workloads
+
+OADP supports backup and restore operations for containerized workloads. 
+
+| VolumeMode | FSB - restic | FSB - kopia | CSI | CSI DataMover (ii) | Volume Snapshot |
```

Comment:
Calling it "Volume Snapshot" is confusing, since VolumeSnapshot is the CR created for CSI snapshots. Maybe "native snapshots (VolumeSnapshotter plugin)? If that's too long, then just "Native snapshots" and a note below the chart saying that native snapshots are only supported if there's an included VolumeSnapshotter plugin.

Reply:
Oh, there's already a note below for the scope of this column. In that case, either "Native snapshots" or "Native snapshots (VolumeSnapshotter plugin)" for the heading here, and use the same wording beginning the note below.

Reply:
Cloud Provider Native Snapshots

Reply:
nice, thank you both!